interfaces/builtin/lvm_support.go

```diff
@@ -0,0 +1,75 @@
+// -*- Mode: Go; indent-tabs-mode: t -*-
+
+/*
+ * Copyright (C) 2020 Canonical Ltd
+ *
+ * This program is free software: you can redistribute it and/or modify
+ * it under the terms of the GNU General Public License version 3 as
+ * published by the Free Software Foundation.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ * GNU General Public License for more details.
+ *
+ * You should have received a copy of the GNU General Public License
+ * along with this program.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ */
+
+package builtin
+
+const lvmSupportSummary = `allows access to LVM logical volume devices`
+
+const lvmSupportBaseDeclarationPlugs = `
+  lvm-support:
+    allow-installation: false
+    deny-auto-connection: true
+`
+
+const lvmSupportBaseDeclarationSlots = `
+  lvm-support:
+    allow-installation:
+      slot-snap-type:
+        - core
+    deny-auto-connection: true
+`
+
+const lvmSupportConnectedPlugAppArmor = `
```

Comment:
As a sidenote, the core{,18,20} do not ship any of the pv*, vg*, lv*, tools, so those must be shipped with a snap if needed.

Reply:
Yes, I should probably add comment / doc to that effect, thx! :+1: